share/src/main/java/io/zensoft/share/model/diesel/RequestStatus.java

```diff
@@ -0,0 +1,7 @@
+package io.zensoft.share.model.diesel;
+
+public enum RequestStatus {
```

Comment:
May be its better to name it in a singular form

Reply:
ok, done.